key-attestation/java-example/README.md

```diff
@@ -15,9 +15,11 @@ This is a example java of how to verify **Fortanix DSM Key Attestation Statement
 The test code under [VerifyTest.java](src/test/java/com/fortanix/keyattestationstatementverifier/VerifyTest.java)
 shows how to properly verify the  **Fortanix DSM Key Attestation Statement** certificate:
 
+Online check
+- `verifyStatementFullCheck`: Verify given `KeyAttestationResponse` in a JSON file, the `Fortanix Attestation and Provisioning Root CA` is downloaded from https://pki.fortanix.com in runtime. **Note**: This test is turned off since CRL and PKI server is not ready when this example code is created and certificates for testing are signed by fake CA.
```

Comment:
Do you think it would be worthwhile to update the test once our PKI server is ready?

Reply:
Yes, we need to wait for production server online then get a real statement to be added to test.
Created issue for tracking: https://github.com/fortanix/user-assurance/issues/3